proto/stream_plan.proto

```diff
@@ -50,11 +50,9 @@ message FilterNode {
 message MaterializeNode {
   uint32 table_id = 1;
```

Comment:
You will need to remove this in next PR and make changes to stream_graph.rs, meta/stream/mod.rs to add the table_id to the table.